### DISCUSSION TITLE
[FEATURE] Add a built-in "deadline" field in the GitHub Project

### DISCUSSION THREAD

**Discussion by hello-ehsan** [community/community#38748] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It would be great if the GitHub project has a built-in "deadline" or "follow-up" field that can send a reminder email and the date become red when the deadline is close or passing! Can be accessed from the GitHub project.

Currently, GitHub Board has the "Date" custom field. But something like the "Deadlines" or "Follow-up" field would be much better. This feature might have other sub-features too -
- Assigned person will get a notification through email/GitHub when the deadline is set
- He/she will get a follow-up message/notification/email before X days of the deadline
- We can set the timeline of an issue from 1st Jan to 10th Jan or simply one date (end time) with an alarming/notifying feature

**Comment by ossanna16** [comment]
Hi there @hello-ehsan and welcome to our community! Thank you for the feature request!

**Reply by hello-ehsan** [reply]
@ossanna16 
Thanks!
Recently GitHub introduces the Roadmap feature. Do you have any plan to send notification to email (specially to Slack) when it crosses the deadline?

**Reply by ossanna16** [reply]
@hello-ehsan Great question, unfortunately I currently don't have any information on that.
